composer.json

```diff
@@ -25,15 +25,14 @@
         }
     },
     "require": {
-        "php": ">=5.6.0",
-        "guzzlehttp/guzzle": "^6.2",
-        "illuminate/support": "^5.0",
-        "nesbot/carbon": "~1.21",
-        "predis/predis": "^1.1"
+        "php": "^7.2",
+        "guzzlehttp/guzzle": "^6.4",
+        "nesbot/carbon": "^2.27",
+        "illuminate/support": "^6.16"
```

Comment:
to keep it backward compatible with previous Laravel versions, use
```
"illuminate/support": "^5.0 || ^6.0",
"nesbot/carbon": "^1.0 || ^2.0",
```

Reply:
Awesome insight, I have updated the requested changes with this commit 355f07868c90706838808f976c17c629306cc221